aries-test-harness/features/didcomm-v2.feature

```diff
@@ -0,0 +1,48 @@
+@DIDComm-V2 @UsesCustomParameters
+Feature: DIDComm V2 Establishing Connections (with Issue Credential example)
+
+   @T001-DIDComm-V2
+   Scenario: Establish a connection between two agents using DIDComm V2
+      Given we have "2" agents
+         | name | role      |
+         | Acme | responder |
+         | Bob  | requester |
+
+      And "Acme" is running with parameters "{"mime-type":["didcomm/v2"]}"
+      And "Bob" is running with parameters "{"mime-type":["didcomm/v2"]}"
+
+      When "Acme" creates a didcomm v2 invitation
+      And "Bob" accepts the didcomm v2 invitation from "Acme"
+
+      # Bob must send a message to acme
+      When "Bob" requests mediation from "Acme"
+
+      And "Acme" has received a didcomm v2 message from "Bob" and created a connection
+
+   @T001.1-RFC0453-DIDComm-V2 @RFC0453 @AIP20 @RFC0593 @critical @AcceptanceTest @DIDExchangeConnection @CredFormat_JSON-LD @Schema_DriversLicense_v2 @ProofType_Ed25519Signature2018 @DidMethod_key
+   Scenario Outline: Issue a JSON-LD Ed25519Signature2018 credential with the Holder beginning with a proposal, using DIDComm v2 for the connection
```

Comment:
If the point of this test is to issue a cred with two actors with a didcimm-v2 connection, I think we can shorted this up a bit. The When clause should always be the actions taken directly related to the point of the test. So it could be slightly simplified like this, 
```
Given "2" agents
...
And "Acme" is running with parameters "{"mime-type":["didcomm/v2"]}"
And "Bob" is running with parameters "{"mime-type":["didcomm/v2"]}"
And "Acme" is ready to issue a "json-ld" credential
And "Acme and "Bob" have a didcimm v2 connection 
When "Bob" proposes a "json-ld" credential to "Acme" with <credential_data>
And "Acme" offers the "json-ld" credential
And "Bob" requests the "json-ld" credential
And "Acme" issues the "json-ld" credential
And "Bob" acknowledges the "json-ld" credential issue
Then "Bob" has the "json-ld" credential issued
```